test/spec/adapters/districtm_spec.js

```diff
@@ -0,0 +1,293 @@
+        it('For loading file ', ()=>{
+            expect(!window.hb_dmx_res).to.equal(true);
+        })
+
+
+    })
+
+
+    describe('check for library do exists', ()=>{
+        it('library was not loaded', ()=>{
+
+            expect(!window.hb_dmx_res).to.equal(true);
+        })
+
+        it('library is now available', ()=>{
+            activated();
+
+            expect(!!window.hb_dmx_res).to.equal(true);
+
+        })
+    })
+
+
+    describe('Check if size get clean', ()=>{
+        beforeEach(()=>{
+            activated();
+        })
+        it('size clean up using fixe size', ()=>{
+            activated();
+
+            expect(window.hb_dmx_res.auction.fixSize([728, 90])[0]).to.equal(['728x90'][0]);
+            expect(window.hb_dmx_res.auction.fixSize([[300, 250], [300,600]]).toString()).to.equal(['300x250', '300x600'].toString());
+
+        })
+    })
+
+    describe('Check call bids return no errors', ()=>{
+        let districtm;
+        beforeEach(()=>{
+            districtm = new Adaptor();
+        });
+        it('check value push using cal bids', ()=>{
+            let obj = districtm.callBids(PREBID_PARAMS);
+            obj.should.have.property('bidderCode');
+            obj.should.have.property('requestId');
+            obj.should.have.property('bidderRequestId');
+            obj.should.have.property('start');
+            obj.should.have.property('bids');
+
+        })
+        it('check if value got pass correctly for DM params', ()=>{
+            let dm = districtm.callBids(PREBID_PARAMS).bids.map( bid => bid);
+            dm.forEach( a =>{
+                a.should.have.property('bidder');
+                a.should.have.property('requestId');
+                a.should.have.property('bidderRequestId');
+                a.should.have.property('placementCode');
+                a.should.have.property('params');
+                a.should.have.property('sizes');
+                expect(a.bidder).to.equal('districtmDMX');
+                expect(a.placementCode).to.equal('golden');
+                expect(a.params.placement).to.equal(109801);
+            })
+
+
+        })
+    })
+
+    describe('Run prebid definitions !', ()=>{
+         let districtm;
+        beforeEach(()=>{
+            districtm = new Adaptor();
+        })
+
+        it('Run and return send bids', ()=>{
+            let sendBids  =  districtm.sendBids(PREBID_PARAMS);
+            sendBids.forEach(sb =>{
+
+                expect(sb.sizes.toString()).to.equal([[300, 250], [300, 600]].toString());
+            })
+
+        })
+
+
+    })
+
+    describe('HandlerRes function test', ()=>{
+       let districtm;
+
+        beforeEach(()=>{
+            districtm = new Adaptor();
+        })
+
+        it('it\'s now time to play with the response ...', ()=>{
+            let result = districtm.handlerRes(PREBID_RESPONSE(), PREBID_PARAMS);
+            _each(result, function(k, v){
+                console.log(11, `${k} value is ${v}`);
```

Comment:
In the `test` directory we try to avoid logging to the console to avoid having non-build related activity displayed during the build process. Please convert this to an assertion if it makes sense or remove the line